server/src/main/java/dev/slimevr/tracking/processor/skeleton/LegTweaks.java

```diff
@@ -810,9 +812,9 @@ private void correctFootRotations() {
 		if (bufferHead == null || bufferHead.getParent() == null)
 			return;
 
-		// if there is a foot tracker for a foot don't correct it
-		if (skeleton.leftFootTracker != null || skeleton.rightFootTracker != null)
-			return;
+		// boolean for if there is a foot tracker
+		boolean footTrackers = skeleton.leftFootTracker != null
+			|| skeleton.rightFootTracker != null;
```

Comment:
Could you make it 2 separate variables, one per foot, in order to support any ankle + foot setup?

Reply:
for sure! while I'm at it all make toe snap also do that also